container_api/src/config.rs

```diff
@@ -159,31 +153,31 @@ hash = "Qm328wyq38924y"
 #[test]
 fn test_load_complete_config() {
     let toml = r#"
-[[agent]]
+[[agents]]
 id = "test agent"
 name = "Holo Tester"
 key_file = "holo_tester.key"
 
-[[dna]]
+[[dnas]]
 id = "app spec rust"
 file = "app-spec-rust.hcpkg"
 hash = "Qm328wyq38924y"
 
-[[instance]]
+[[instances]]
 id = "app spec instance"
 dna = "app spec rust"
 agent = "test agent"
-[instance.logger]
+[instances.logger]
 type = "simple"
 file = "app_spec.log"
-[instance.storage]
+[instances.storage]
 type = "file"
 path = "app_spec_storage"
 
-[[interface]]
+[[interfaces]]
 type = "websocket"
 port = 8888
-[[interface.instance]]
+[[interfaces.instances]]
```

Comment:
This is one case where plurals makes it clearer -- originally I thought it was one-to-one because it was `interface.instance`

Reply:
gotcha 👍